### ISSUE TITLE
Version Packages

### ISSUE THREAD

**Issue by AngangGuo** [mxschmitt/playwright-go#190]
@kumaraditya303 
Thanks for your great work to commit the "roll to 1.14" to match the latest version of Playwright.
Will you able to version it or publish the release?

**Comment by kumaraditya303** [comment]
> Thanks for your great work to commit the "roll to 1.14" to match the latest version of Playwright.


Thanks :)

I am still implementing the new locators API of version v1.14 of playwright, so perhaps first version v1.13 can be released and then v1.14 can be released.

cc @mxschmitt for his thoughts on next release.

**Comment by mxschmitt** [comment]
Yeah we can definitely publish a new release, its just important that we stay in the 0.x area, so we end up in a weird release like 0.1400.

**Comment by AngangGuo** [comment]
Sounds good. Great thanks 👍
